lib/ecom/types.ts

```diff
@@ -67,6 +67,10 @@ export type EcomApi = {
     getAllCategories: () => Promise<Collection[]>;
     getCategoryBySlug: (slug: string) => Promise<CollectionDetails | undefined>;
     getOrder: (id: string) => Promise<OrderDetails | undefined>;
+    getOrders: () => Promise<{
+        orders: OrderDetails[];
+        metadata?: orders.CursorPagingMetadata;
```

Comment:
why do we need this?

Reply:
it's something return by the SDK and contains information needed for the pagination (`count`, `hasNext`). I'll change it to return the same info as `queryProducts` does - `{items: [], totalCount: 0}`